presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -260,6 +263,8 @@ protected void setup(Binder binder)
         // analyze properties
         binder.bind(AnalyzePropertyManager.class).in(Scopes.SINGLETON);
 
+        binder.bind(NodeStatusServiceConfig.class).in(Scopes.SINGLETON);
```

Comment:
change this to `configBinder(binder).bindConfig(NodeStatusServiceConfig.class)` - this will make it actually read configuration properties. As of now the class will be initialized like any other class

Reply:
done